db.js

```diff
@@ -0,0 +1,23 @@
+const { MongoClient } = require("mongodb");
```

Comment:
You will need to `require` the schema variables exported by the schema file:

```suggestion
const { MongoClient } = require("mongodb");
const { userSchema, roomSchema, messageSchema } = require("./schema");
```